common/calclib/fixed_point_utils.h

```diff
@@ -53,4 +52,19 @@ int64_t int_cast(T&& arg) {
     return fp_cast<int64_t>(std::forward<T>(arg));
 }
 
+template <typename U, typename T>
+void narrow_down(U& lhs, T& rhs) {
+    FIXED_POINT_ASSERT((lhs >= 0) && (rhs >= 0), "narrow_down: wrong args");
+    using narrow_t = typename std::conditional<(U::integer_digits < T::integer_digits), U, T>::type;
+    using comp_t   = typename std::conditional<(U::integer_digits > T::integer_digits), U, T>::type;
+    constexpr comp_t narrow_max = std::numeric_limits<narrow_t>::max();
+    
+    auto& lhs_data = const_cast<typename U::rep&>(lhs.data());
+    auto& rhs_data = const_cast<typename T::rep&>(rhs.data());
+    while((static_cast<comp_t>(lhs) > narrow_max) || (static_cast<comp_t>(rhs) > narrow_max)) {
```

Comment:
Let's move to common code-style: we put have space after `for`/`while`/`if`. There is no need to refactor all code, but new code should conform code-style.
ref: https://github.com/GolosChain/golos/wiki/Example-code-style